tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -808,23 +864,38 @@ def main():
     if use_corked_io == True:
         print("Corked IO is on")
 
+    results = {'tests_ran': 0}
+
     try:
-        resume_test(host, port, test_ciphers, fips_mode, no_ticket=True)
-        resume_test(host, port, test_ciphers, fips_mode)
-        handshake_test(host, port, test_ciphers, fips_mode)
-        client_auth_test(host, port, test_ciphers, fips_mode)
-        sigalg_test(host, port, fips_mode)
-        sigalg_test(host, port, fips_mode, use_client_auth=True, no_ticket=True)
-        elliptic_curve_test(host, port, libcrypto_version, fips_mode)
-        elliptic_curve_fallback_test(host, port, fips_mode)
-        handshake_fragmentation_test(host, port, fips_mode)
-        ocsp_stapling_test(host, port, fips_mode)
-        cert_type_cipher_match_test(host, port, libcrypto_version)
-        multiple_cert_type_test(host, port, libcrypto_version)
-        multiple_cert_domain_name_test(host, port)
+        for tls13_flag in [False, True]:
+            # options are additional arguments that eventually gets passed to try_handshake()
+            options = dict(
+                results=results,
+                tls13_flag=tls13_flag,
+                debug_cmds=args.debug,
+                simulate_tests=args.dryrun,
+            )
+            if tls13_flag:
+                print("\n\n------------- Testing with TLS1.3 Flag -------------\n\n")
+
+            resume_test(host, port, test_ciphers, fips_mode, no_ticket=True, **options)
```

Comment:
Same spirit as the last comment, we want to refactor all this anyway.

`options` here may be better called `handshake_options`, which makes it a little clearer where the options are actually used. Although having a `class IntegrationTest(object)` which gets configured, and then all these tests are either methods, or subclasses, or whatever, would be more explicit. This prevents `**kwargs` from being passed around.

But again, these changes are baked in to the new framework. Pragmatically, whatever adds more test coverage here is fine with me.

Reply:
I didn't want to refactor more than this PR needed to be, but I'm totally passing a class that encapsulates all the options would be the approach